src/test/java/io/pravega/connectors/flink/utils/SetupUtils.java

```diff
@@ -274,25 +302,29 @@ public void start() throws Exception {
 
             this.inProcPravegaCluster = InProcPravegaCluster.builder()
                     .isInProcZK(true)
+                    .secureZK(enableTls)
```

Comment:
Since this method is for enabling security for Zookeeper, using the `enableTls` property here looks a bit odd from a readability perspectively, even though it is logically correct. At the same time, using an "explaining variable" or an explicit condition is probably too complicated for what is being done here. Maybe a comment is in order.

Reply:
I have added an inline comment to explain what it means.